### ISSUE TITLE
Find network interface by name

### ISSUE THREAD

**Issue by mdolian** [k1LoW/awspec#190]
It doesn't look like this has been implemented.  I attempted to add code that would search by name but it doesn't appear to be working.  I'd be happy to do this work but I need a little guidance.  Should I only have to modify the find_network_interface in the ec2 finder or is there something else I have to update to get this working?

```
      def find_network_interface(interface_id)
         res = ec2_client.describe_network_interfaces({
                                                          filters: [
                                                            {
                                                              name: 'network-interface-id',
                                                              values: [interface_id]
                                                            }
                                                          ]
                                                        })
        resource = res.subnets.single_resource(interface_id)
        return resource if resource
        res = ec2_client.describe_network_interfaces({
                                                 filters: [{ name: 'tag:Name', values: [interface_id] }]
                                               })
        res.network_interfaces.single_resource(interface_id)
      end
```

**Comment by k1LoW** [comment]
Thank you for your contribution!!!
Nice work!!

You change only find_network_interface(), maybe.

( And `s/subnets/network_interfaces/`.  )

Regards.

**Comment by k1LoW** [comment]
Closed by #191
